src/app/ssmt/calculate-losses.service.ts

```diff
@@ -234,10 +234,10 @@ export class CalculateLossesService {
       highPressureHeader: 0,
       mediumPressureHeader: 0,
       lowPressureHeader: 0,
-      condensingTurbineEfficiencyLoss: 0,
-      highToMediumTurbineEfficiencyLoss: 0,
-      highToLowTurbineEfficiencyLoss: 0,
-      mediumToLowTurbineEfficiencyLoss: 0,
+      condensingTurbineEfficiencyLoss: -1,
```

Comment:
I like the idea here but it's gonna cause some issues in other areas of calculations if we set the values to -1 like calculate totalTurbineLosses(); 

I suggest, modifying the `SSMTLosses` object by adding four booleans that you use for show/hiding turbine. Use those booleans in the `*ngIf` statement.

They will end up matching the `.useTurbine` booleans of the `turbineInput` object in the `calculateLosses()` function.

I'm happy to hop on a call to explain further.